src/docs/backend/control-silo.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "Control Silo"
+---
+
+Within the Control Silo are features that allow us to provide backwards compatibility for both customer API usage, and integrations
+
+# API Gateway
+
+Prior to our multi-region offering, all API traffic from customers would be sent to [sentry.io](http://sentry.io) . We need to continue supporting these URLs despite some customers residing in the `us` region and others in the `de`region. The API Gateway functionality of control silo will introspect requests and proxy requests to the appropriate region based on path parameters in the request. Requests can be routed via:
+
+- Organization slug or id
+- Sentry App Installation UUID
+- Sentry App Slug
+- DSN host
+- A static list of URL names (region pin list)
+
+<Alert level="info">
+💡 Requests going to [sentry.io](http://sentry.io) do not meet data residency requirements. Customers will need to use region domains in order to not have requests go to the US.
```

Comment:
```suggestion
💡 Requests going to sentry.io do not meet data residency requirements. Customers will need to use region domains in order to not have requests go to the US.
```

Since the preview doesn't want to link this